doc/user/content/concepts/clusters.md

```diff
@@ -28,8 +28,15 @@ to be associated with a cluster:
 You choose the size of your cluster (`25cc`, `50cc`, `100cc`, etc.) based on
 the resource requirements of your workload. Larger clusters have more compute
 resources available and can therefore process data faster and handle larger data
-volumes. You can resize a cluster at any time, even while the cluster is
-running, to respond to changes in your workload.
+volumes.
+
+You can resize a cluster to respond to changes in your workload.
+
+{{< note >}}
+
+Resizing incurs downtime.
```

Comment:
```suggestion
Resizing incurs downtime, as it requires all objects in the cluster to hydrate.
In the future, it will be possible to resize clusters with no downtime {{% gh 20010 %}}.
```